manifests/j/JetBrains/IntelliJIDEA/Community/203.8084.24/JetBrains.IntelliJIDEA.Community.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: JetBrains.IntelliJIDEA.Community
+PackageVersion: 203.8084.24
+PackageLocale: en-US
+Publisher: JetBrains

```

Comment:
```suggestion
Publisher: JetBraoms s.r.o.
```
![image](https://user-images.githubusercontent.com/11600822/120534673-063ed500-c3da-11eb-8f58-f5135b030479.png)

Reply:
All the other version of intellij and also other jet brains things have publisher as 'Jetbrain ' if I update this it would become odd one out

Reply:
> ```suggestion
> Publisher: JetBrains s.r.o.
> ```
> ![image](https://user-images.githubusercontent.com/11600822/120534673-063ed500-c3da-11eb-8f58-f5135b030479.png)
> 

All the other version of intellij and also other jet brains things have publisher as 'Jetbrains ' if I update this it would become odd one out